Documentation theory guide (#427)
lethe-cfd/lethe: doc/source/theory/theory.rst

############
Theory
############

> Lethe uses the Finite Element Method (FEM) to solve multiple partial differential equations related to fluid dynamics, the most important one being the incompressible Navier-Stokes equations. The aim of this theory guide is to provide information on the exact equations solved within Lethe and give basic indications on the finite elements strategies used (e.g. stabilization) to solve them. This theory guide is not intended to be an introduction to FEM nor the deal.II library. We expect the reader to have a working knowledge of FEM. For more information on FEM we refer the reader to the exquisite `video capsules <https://www.math.colostate.edu/~bangerth/videos.html>`_ created by Wolfgang Bangerth or to books on the topic such as the book by Larson [1], which is very accessible. To learn the basics of the deal.II library, we refer the reader to the `deal.II tutorials <https://www.dealii.org/current/doxygen/deal.II/Tutorial.html>`_.

Reviewer: ```suggestion
Lethe uses the Finite Element Method (FEM) to solve multiple partial differential equations related to fluid dynamics, the most important one being the incompressible Navier-Stokes equations. The aim of this theory guide is to provide information on the exact equations solved within Lethe and give basic indications on the finite element strategies used (e.g. stabilization) to solve them. This theory guide is not intended to be an introduction to FEM nor the deal.II library. We expect the reader to have a working knowledge of FEM. For more information on FEM we refer the reader to the exquisite `video capsules <https://www.math.colostate.edu/~bangerth/videos.html>`_ created by Wolfgang Bangerth or to books on the topic such as the book by Larson [1], which is very accessible. To learn the basics of the deal.II library, we refer the reader to the `deal.II tutorials <https://www.dealii.org/current/doxygen/deal.II/Tutorial.html>`_.
```

> To solve the flow of granular matter, Lethe uses the Discrete Element Method (DEM). This method is clearly not as well established as the FEM. The DEM component of Lethe is, however, fully described in [2]. There are also multiple review articles which give an introduction to this method. Our research group has written on such review article on DEM [3] and how it can be applied to complex chemical engineering problems [4]. There are multiple other review articles that give a good introduction to DEM such as the one by Bertrand [5] or Zhu [6]. There is also a great book on the topic of DEM and its coupling with CFD by Norouzi et al. [7].

Reviewer 2: I'm not sure about the sentence that starts with "Our research group". I tried reformulating it but could you tell me what you meant by that sentence?

> To solve the flow of granular matter, Lethe uses the Discrete Element Method (DEM). This method is clearly not as well established as the FEM. The DEM component of Lethe is, however, fully described in [2]. There are also multiple review articles which give an introduction to this method. Our research group has written on such review article on DEM [3] and how it can be applied to complex chemical engineering problems [4]. There are multiple other review articles that give a good introduction to DEM such as the one by Bertrand [5] or Zhu [6]. There is also a great book on the topic of DEM and its coupling with CFD by Norouzi et al. [7].

Reviewer: ```suggestion
To solve the flow of granular matter, Lethe uses the Discrete Element Method (DEM). This method is clearly not as well established as the FEM. The DEM component of Lethe is, however, fully described in [2]. There are also multiple review articles which give an introduction to this method. Our research group has written a review article on DEM [3] and its application to complex chemical engineering problems [4]. There are multiple other review articles that give a good introduction to DEM such as the one by Bertrand [5] or Zhu [6]. There is also a great book on the topic of DEM and its coupling with CFD by Norouzi et al. [7].
```
